bin/webpack-dev-server.js

```diff
@@ -294,6 +296,25 @@ function processOptions(wpOpt) {
 	if(argv["open"])
 		options.open = true;
 
+	// Kind of weird, but ensures prior behavior isn't broken in cases
+	// that wouldn't throw errors. E.g. both argv port and options port
+	// were specified, but since argv port is 8080, options port will be
+	// used instead.
+	options.port = argv.port === DEFAULT_PORT ? options.port || argv.port : argv.port;
```

Comment:
Aha, thanks for catching that. I didn't account for the case where `options.port` is defined but `argv.port` is not (I forgot since `argv` used to have a default).

What do you think of just changing it to:
```javascript
options.port = argv.port === DEFAULT ? (options.port || argv.port) : (argv.port || options.port)
```
? This should account for all cases I believe.